pkg/operator/ceph/operator.go

```diff
@@ -101,6 +103,23 @@ func (o *Operator) cleanup(stopCh chan struct{}) {
 	o.clusterController.StopWatch()
 }
 
+func (o *Operator) updateOperatorLogs() error {
+	rookLogLevel, err := k8sutil.GetOperatorSetting(o.context.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_LOG_LEVEL", "INFO")
+	if err != nil {
+		logger.Warningf("failed to load ROOK_LOG_LEVEL. Defaulting to INFO. %v", err)
+		rookLogLevel = "INFO"
+	}
+
+	log, err := capnslog.ParseLevel(rookLogLevel)
+	if err != nil {
+		return errors.Errorf("failed to load ROOK_LOG_LEVEL %q. %v", log, err)
+	}
+
+	logLevel = log
```

Comment:
Actually, do we even need to remember the log level in this package var? Every time this code is triggered, we just look it up again with `k8sutil.GetOperatorSetting()` so it seems like we don't need this.
```suggestion
```

Reply:
right, we don't need this. Removed